### PR TITLE
Astro 2/13 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/configure-worker-queues.md [hunk 1]
```diff
@@ -59,6 +59,8 @@ The following table lists all available worker types on Astro Deployments.
 | A20         | 4    | 8GiB   | 10 GiB            | 20                       | 60                   |
 | A40         | 8    | 16GiB  | 10 GiB            | 40                       | 120                  |
 | A60         | 12   | 24GiB  | 10 GiB            | 60                       | 180                  |
+| A120        | 24   | 48GiB  | 10 GiB            | 120                      | 360                  |
+| A160        | 32   | 64GiB  | 10 GiB            | 160                      | 480                  |
```
Comment (jwitz) [inline astro/configure-worker-queues.md:63]:
```suggestion
```

File: astro/release-notes.md [hunk 2]
```diff
@@ -21,6 +21,34 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+This feature is in [Private Preview](https://docs.astronomer.io/astro/feature-previews). Please reach out to your customer success manager to enable this feature.
+
+:::
+
+The new **Dashboards** page includes a suite of dashboards that you can use to asses the performance of Deployments and DAGs across your entire Organization. Each dashboard focuses on a different aspect of your data pipelines to show you opportunities for cost and performance improvements. You can additionally configure Astro to send you alerts when a given metric reaches a specific threshold. See [Reporting dashboards](reporting-dashboard.md) for summaries of each available dashboard.
+
+### New worker types
+
+Astro Hosted Deployments now support A120 and A160 workers, which include enough CPU and memory to handle the most resource-intensive tasks in your DAGs. See [Astro Hosted resource reference](resource-reference-hosted.md#deployment-resources) for more information about each worker type.
+
```
Comment (jwitz) [inline astro/release-notes.md:39]:
```suggestion
```

File: astro/release-notes.md [hunk 3]
```diff
@@ -21,6 +21,30 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+The new **Dashboards** page includes a suite of dashboards that you can use to asses the performance of Deployments and DAGs across your entire Organization. Each dashboard focuses on a different aspect of your data pipelines to show you opportunities for cost and performance improvements. You can additionally configure Astro to send you alerts when a given metric reaches a specific threshold. See [Reporting dashboards](reporting-dashboard.md) for summaries of each available dashboard.
+
+### Additional improvements
+
+- When you submit a support request from the Cloud UI, you must now define an **Active Engagement Period** when you or a member of your team can engage with a member of Astronomer support. 
+- Workspace Members can now access the **Clusters** view in the Airflow UI for a Deployment.
+
+### Bug fixes
+
+- Fixed an issue where network connections between clusters could occasionally be disrupted.
```
Comment (lzdanski) [inline astro/release-notes.md:43]:
```suggestion
- Fixed an issue where network connections between clusters could be disrupted occasionally.
```

Infinitive split - This feels like such a pedantic nit, feel free to take or leave it!

**Review by lzdanski — APPROVED** [review]
Super minor thing, overall LGTM

File: astro/release-notes.md [hunk 4]
```diff
@@ -21,6 +21,30 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+### New Astro reporting dashboards show metrics for Deployments across your Organization
+
+:::warning
+
+This feature is in [Private Preview](https://docs.astronomer.io/astro/feature-previews). Please reach out to your customer success manager to enable this feature.
+
+:::
+
+The new **Dashboards** page includes a suite of dashboards that you can use to asses the performance of Deployments and DAGs across your entire Organization. Each dashboard focuses on a different aspect of your data pipelines to show you opportunities for cost and performance improvements. You can additionally configure Astro to send you alerts when a given metric reaches a specific threshold. See [Reporting dashboards](reporting-dashboard.md) for summaries of each available dashboard.
```
Comment (jwitz) [inline astro/release-notes.md:34]:
```suggestion
The new **Dashboards** page includes a suite of dashboards that you can use to asses the performance of Deployments and DAGs across your entire Organization. Each dashboard focuses on a different aspect of your data pipelines to show you opportunities for cost and performance improvements. You can additionally configure Astro to send you alerts when a given metric reaches a specific threshold. See [Reporting dashboards](https://docs.astronomer.io/astro/reporting-dashboard) for summaries of each available dashboard.
```

**Pull request merged**